src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx

```diff
@@ -148,10 +149,10 @@ export const CollapsibleNavTop = ({
           />
         </EuiFlexItem>
       </EuiFlexGroup>
-      {currentNavGroup?.title && (
+      {currentNavGroup?.title && collapsibleNavHeaderRender && (
         <>
           <EuiSpacer />
-          <EuiText>{currentNavGroup?.title}</EuiText>
+          {collapsibleNavHeaderRender()}
```

Comment:
```suggestion
          {collapsibleNavHeaderRender ? collapsibleNavHeaderRender() : <EuiText>{currentNavGroup?.title}</EuiText>}
```
We should take the case when workspace is not enabled into consideration.

Reply:
We need to render the `currentNavGroup?.title` when we are not in a workspace.(data administration / settings and setup)